OpenWPM.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+    "folders": [
+        {
+            "path": "openwpm"
+        }
+    ]
+}
```

Comment:
Could you please remove this file and add it to the .gitignore explaining what it is